### DISCUSSION TITLE
Is it possible for someone to find out the specific account used to exported a server?

### DISCUSSION THREAD

**Discussion by madsinn** [Tyrrrz/DiscordChatExporter#675] [Q&A]
So recently, I exported some of a server's chatlogs and posted them online anonymously. I'm worried that someone might be able to go through the and find out the account that I did it from, whether it be from some sort of security oversight or something. 

Is this a real concern? Or is it not possible? 

Thank you for your time. :)

**Comment by 96-LB** [comment]
As far as I know, no data about the account used to export is included anywhere in the file.

**Reply by madsinn** [reply]
Okay, thank you!
